src/views/start/index.jsx

```diff
@@ -168,7 +181,8 @@ export default class Start extends Component {
     return (
       <div className="Start">
         <div>
-          Query Parameter :
+          Query parameter:
+          &nbsp;
```

Comment:
In react you can write {" "} ... But one single space should be possible by just placing one behind the colons. Not 100% sure

Reply:
This works great 👍